lib/utils/transformpipeline/transforms.py

```diff
@@ -389,6 +389,11 @@ def transform_value(self, entry: dict) -> dict:
             if not entry['authors'].strip('. ').endswith(" et al"): # if it does not already finishes with " et al.", add it
                 entry['authors'] += ' et al'
 
+            # Strip all double quotes from the authors value to prevent
+            # unclosed quotes from causing errors in table readers when reading
+            # the transformed metadata TSV
+            entry['authors'] = entry['authors'].replace('"', '')
```

Comment:
Embedded quotes in column values in TSV/CSV are fine and well-supported. We shouldn't have to do this mangling, and it's a band-aid anyway as any other column could also contain unbalanced quotes.

Why is our CSV emitter not properly escaping the quotes in our values?

Reply:
Good point! Looking at this again, in the output from `transform-gisaid`, the authors values with the unclosed `"` are double quoted. I believe this is the default behavior for `csv.DictWriter`. 

Then looking at the output from `join-metadata-and-clades`, these are still double quoted. This is the expected default behavior for `pandas.to_csv`. 

So, it appears that the issue is purely in the acknowledgements.tsv which would make this an Auspice issue.

Reply:
🙈